inst/rmarkdown/templates/thesis/skeleton/01-chap1.Rmd

```diff
@@ -174,10 +174,17 @@ flights %>% filter(month == 3, day == 3) %>%
 
 ## Additional resources
 
-- _Markdown_ Cheatsheet - <https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet>
+- _Markdown_ Cheatsheet - <https://github.com/rstudio/cheatsheets/raw/master/rmarkdown-2.0.pdf>
```

Comment:
This is more focused on R Markdown than just Markdown as the original cheatsheet had and is still active. Maybe it's better to replace the link below with this R Markdown cheatsheet that you suggest instead?